draft-ietf-rats-architecture.md

```diff
@@ -672,26 +672,60 @@ consumes the corresponding conceptual messages as defined in this document.
 
 # Trust Model {#trustmodel}
 
+## Relying Party
+
 The scope of this document is scenarios for which a Relying Party
 trusts a Verifier that can appraise the trustworthiness of
 information about an Attester.  Such trust might come by the Relying
 Party trusting the Verifier (or its public key) directly, or might
 come by trusting an entity (e.g., a Certificate Authority) that is
-in the Verifier's certificate chain.  The Relying Party
-might implicitly trust a Verifier (such as in the Verifying Relying
-Party combination).  Or, for a stronger level of security, the
-Relying Party might require that the Verifier itself provide
+in the Verifier's certificate chain.
+
+The Relying Party
+might implicitly trust a Verifier, such as in a Verifying Relying
+Party combination where a Verifier and a Relying Party are combined.
+Or, for a stronger level of security, the
+Relying Party might require that the Verifier first provide
 information about itself that the Relying Party can use to assess
 the trustworthiness of the Verifier before accepting its Attestation Results.
 
-The Endorser and Verifier Owner may need to trust the Verifier
-before giving the Endorsement and Appraisal Policy to it.
-Such trust can also be established directly or indirectly,
-implicitly or explicitly. One explicit way to establish such trust
-may be the Verifier first acts as an Attester and creates Evidence about itself to be consumed by the
-Endorser and/or Verifier Owner as the Relying Parties.
-If it is accepted as trustworthy, then they can provide Endorsements
-and Appraisal Policies that enable it to act as a Verifier.
+For example, one explicit way for a Relying Party "A" to establish
+such trust in a Verifier "B", would be for B to first act as an Attester
+where A acts as a Verifying Relying Party.  If A then accepts B as
+trustworthy, it can choose to accept B as a Verifier for other Attesters.
+
+Similarly, the Relying Party also needs to trust the Relying Party Owner
+for providing its Appraisal Policy for Attestation Results, and
+in some scenarios the Relying Party might even require that the
+Relying Party Owner attest to it before the Relying Party will accept
+an updated policy. This can be done similarly to how a Relying Party
+could establish trust in a Verifier as discussed above.
+
+## Attester
+
+In some scenarios, Evidence might contain sensitive information such as
+Personally Identifiable Information.
+Thus, an Attester must trust entities to which it conveys Evidence, to not
+reveal sensitive data to unauthorized parties.
```

Comment:
(as an aside: is there an architectural requirement on the Evidence container that PII be identified as such?)

Reply:
(as an aside: that's a complex question, since what's considered PII might vary by jurisdiction and the evidence might be conveyed across jurisdictions)